examples/trpo_bullet_kuka.py

```diff
@@ -0,0 +1,45 @@
+from rllab.algos.trpo import TRPO
```

Comment:
When I run this, I get a Bullet visualizer window with a robot moving very slowly. If I set `n_parallel=2`, I get 3 bullet windows. We need to run `n_parallel=72` sometimes. 

* Is there a way to turn off the pybullet visualizer for all but the plotting thread? I should see no windows if I set `plot=False` (I understand this may be a prexisting GymEnv bug).
* Can we tell pybullet to run faster than real time (i.e. as fast as it can)?

Reply:
I found no way to modify those parameters once the env was registered. Those attributes are made private and are set during env registration. I will trace how it works for current gym env.

Reply:
You can modify the parameters trivially by not using registration and constructing them directly.

`plot=True` controls whether or not env.render() is ever called. The assumption in rllab is that if env.render() is never called, no GUI will be used. This allows rllab to spawn n environments but only 1 GUI, because it only ever called render() on one of the environments it makes.

The bullet envs change their behavior in the constructor, not the first time env.render() is called. This is the root of why this problem is difficult.

We don't want to rewrite large parts of rllab or bullet.

What if all environments are constructed with `render=False`, but when I call BulletEnv.render(), it silently reconstructs its underlying bulletenv into one with render=True?

Reply:
The issue here is that `pybullet` creates a connection (which controls the rendering) during registration, and that object is used when `render()` is called. `render()` does not create the window. I found no way to render it from outside their code without calling the class like so:

`env = KukaGymEnv(renders=True)`

But it creates windows for every instance of that environment being called.
I am not sure how to and if it possible to construct a single window.
Can we meet to discuss more on this?